package.json

```diff
@@ -33,7 +33,7 @@
     "slrc": "./bin/sl-runctl.js"
   },
   "scripts": {
-    "lint": "eslint . && jscs .",
```

Comment:
I'm worried that jscs would find things eslint doesn't, that did happen in my testing, but lets get rid of it for now. I'll keep it around globally, and run by hand every once in a while, just to see if it ever catches anything that eslint missed that is important.